schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/common/DefaultOrganizationService.java

```diff
@@ -83,6 +83,26 @@ public Organization getOrganization(Key id) throws EntityNotFoundException, Ille
         }
     }
 
+    @Override
+    public boolean isOrganizationMember(User user) {
+        for (Organization organization : getOrganizations()) {
+            if (organization.getUsers().contains(user))
+                return true;
+        }
+        return false;
```

Comment:
This is very expensive.  We should be able to query on OrganizationMember with userId and to a asKeyList